src/float.rs

```diff
@@ -2074,10 +2197,22 @@ fn integer_decode_f64(f: f64) -> (u64, i16, i8) {
     (mantissa, exponent, sign)
 }
 
+// FIXME: implement it
+#[cfg(feature = "f128")]
+fn integer_decode_f128(_f: f128) -> (u64, i16, i8) {
+    unimplemented!("radix part of `f128` is wider than `u64`")
```

Comment:
We _could_ deal with this lossily by truncating the radix to its 64 most significant bits. That would need careful documentation, but I think it's better than just panicking, and we can still explore a fuller alternative per #98.